db/migrate/20220810152340_add_parent_child_relationships.rb

```diff
@@ -0,0 +1,9 @@
+class AddParentChildRelationships < ActiveRecord::Migration[7.0]
+  def change
+    create_table :organisation_relationships do |t|
+      t.integer :child_organisation_id, foreign_key: true
+      t.integer :parent_organisation_id, foreign_key: true
+      t.timestamps
```

Comment:
Apparently this is good practice, but current tables in the database seem inconsistent as to whether they have timestamp data - do we want this?

Reply:
I generally keep timestamps on these kinds of join tables. It adds very little overhead, but it's useful to debug when a relationship was created.

Reply:
I think all existing tables have timestamps? Which ones don't?

Reply:
@baarkerlounger Ah you're right, they all have created_at at least - it's the couple that don't have updated_at columns I was thinking of (logs_exports and versions), never mind!

Reply:
`versions` isn't really ours, it comes from https://github.com/paper-trail-gem/paper_trail